pipenv/utils.py

```diff
@@ -946,3 +947,23 @@ def download_file(url, filename):
 
     with open(filename, 'wb') as f:
         f.write(r.content)
+
+
+def need_update_check():
+    mkdir_p(PIPENV_CACHE_DIR)
+    p = os.sep.join((PIPENV_CACHE_DIR, '.pipenv_update_check'))
+    if not os.path.exists(p):
+        return True
+    out_of_date_time = time() - (24 * 60 * 60)
+    if os.path.isfile(p) and os.stat(p).st_mtime <= out_of_date_time:
```

Comment:
I think os.getmtime() would be better for cross platform compatibility?

Reply:
^^ it might be, I actually don't know.  Let me check

Reply:
I believe `st_mtime` is platform-independent but `getmtime` is slightly more readable and I prefer it personally